packages/slice-machine/lib/models/common/Environment.ts

```diff
@@ -13,17 +13,22 @@ export interface UpdateVersionInfo {
   updateAvailable: boolean;
 }
 
-export default interface Environment {
+export default interface BackendEnvironment extends FrontEndEnvironment {
   cwd: string;
+  prismicData: PrismicData;
+  client: DefaultClient | FakeClient;
+}
+
+export interface FrontEndEnvironment {
   userConfig: UserConfig;
-  hasConfigFile: boolean;
   repo?: string;
-  prismicData: PrismicData;
   chromatic?: Chromatic;
   updateVersionInfo: UpdateVersionInfo;
   mockConfig: any;
   framework: Framework;
   baseUrl: string;
```

Comment:
Not sure of what this is.
But i think either it's very similar to `PrismicData.base` and one of them can be removed.
From the only use I could find, i'd say we could just export something like "backendport".
![image](https://user-images.githubusercontent.com/41187899/143854341-a70c3309-99c4-42ba-b04b-305fa15be0c1.png)

Reply:
Done :)

Reply:
Like this

![image](https://user-images.githubusercontent.com/15800180/144431166-bbb751ed-84f2-48c8-a7d5-a08e18e0618b.png)